CharacterCreation/Class-Specific Documentation/Doctor/Doctor Hippocratic Proceedures.txt

```diff
@@ -0,0 +1,337 @@
+
+==Patch Em Up ==
+Adrenaline Reward: 40 Points
+Contagion Reward: 1 Contagion 
+Range:          Touch
+Action:         2 Actions
+    
+    You use your biofoam appllicator to spray the smelly green foam into their
+wounds. The miracle of modern science rapidly reconnects arteries and veins and
+begins to mimic destroyed organs and muscle tissue. It hurts like hell, but it
+will give them a chance to keep it together long enough to get a more permanent
+solution back at base. 
+
+    You must have your Field Applicator drawn in order to use this ability. Your
+patient heals an ammount of health listed on the table below. As your character
+learns more medicine skill, the better they heal. Unless otherwise stated, this
+ability cannot heal a patient above half health.
+
++-----------------+--------------+--------+
+| Med Skill Above | Base Healing |  Dice  |
++-----------------+--------------+--------+
+|       9         |     20       | + 2d10 |
++-----------------+--------------+--------+
+|       19        |     30       | + 3d10 |
++-----------------+--------------+--------+
+|       29        |     35       | + 4d10 |
++-----------------+--------------+--------+
+|       39        |     40       | + 5d10 |
++-----------------+--------------+--------+
+|       49        |     50       | + 6d10 |
++-----------------+--------------+--------+
+
+==Defib ==
+Adrenaline Reward:  80 Points
+Contagion Reward: 2 Contagion
+Range: Touch
+Action: 2 Actions
+
+    You adjust the dials on your biofoam injector to compensate for loss of
+vital fluids and to administer the correct frequency and wattage of tachycardia
+interuptor shock. This is gonna hurt like hell, but they'll live to complain 
+about it at least. Who knows? Maybe they'll even say "Thank you!" this time?
+
+    You must have your Field Applicator drawn in order to use this ability. Only 
+usable on an organic patient that is bleeding out. The person is no longer 
+downed and has 10+2d10 Health Points, rather than having to stabilize the target 
+and then heal them as a separate action.
+
+== Pre-op traige ==
+Adrenaline Reward:  30 Points
+Contagion Reward: 1 Contagion
+Range: 10m
+Action: 1 Action
+
+    You scan the patient and dial your Field Applicator to match the patient's
+unique biochemestry. You also take the time to more closely inspect the patient's
+injuries and prognosticate the battlefield's effect on their condition. Your
+keen eye and experience allows you to better treat your patient's maladies. 
+
+    You must have your Field Applicator drawn in order to use this ability. This 
+trick only works on biologic patients. If you use another procedure on this 
+patient either this turn or the next turn that says you can only heal them up
+to half health, you may ignore that limitation.
+
+==Remote Surgery ==
+Adrenaline Reward:  20
+Range:          20m
+Action:         2 Actions
+
+    Let's take this show on the road! You carefully feed out an omni-glob from
```

Comment:
Proposal: instead of the technobabble, could we make it a short-lived package of medical nanites? (because they disassemble themselves to apply the healing). We could even shoot it through a needle.

Reply:
I'd like to discourage shooting teammates with needles. Can we raise this as a future issue?

Reply:
Before I write that, what do you think about the Doc pre-attaching a remote sugery module to teammates' armor (not using a hard point)? In the same spirit as pre-recording genetic makeups for the Triage ability.